### PR TITLE
Fixing vulnerabilities

### PR DESCRIPTION
**Pull request by tradeshift-larsbojensen**
@Tradeshift/TradeshiftUI

Fixes issue #

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -110,13 +111,15 @@
...
+    "postcss": "^7.0.39",
     "postcss-clean": "^1.2.2",
     "prettier": "1.16.4",
-    "release-it": "^14.11.8",
+    "release-it": "^14.14.3",
     "s3": "^4.4.0",
     "semver": "^5.7.1",
-    "simple-git": "^3.7.1"
+    "simple-git": "^3.7.1",
+    "underscore": "^1.13.3"
```
Comment (esmailpour-hosein) [inline package.json:122]: Why we need undersocre as dev-dep now?

File: package.json [hunk 2]
```diff
@@ -110,13 +111,15 @@
     "lint-staged": "^12.1.2",
     "load-grunt-tasks": "^5.1.0",
     "npm-run-all": "^4.1.1",
-    "octonode": "^0.9.4",
+    "octonode": "^0.10.2",
+    "postcss": "^7.0.39",
```
Comment (esmailpour-hosein) [inline package.json:115]: is it a direct dev dependency or the resolution would be enough?

File: package.json (hunk 1 above)
Comment (tradeshift-larsbojensen) [reply package.json:122]: I am not sure, but the answer lies in the ways of npm. The resolution entry of unbderscore did not do its job until I installed it as a direct (dev) dependency.

File: package.json (hunk 2 above)
Comment (tradeshift-larsbojensen) [reply package.json:115]: For some reason, postcss was not added to node_modules, so a grunt script requiring it failed.

File: package.json (hunk 1 above)
Comment (tradeshift-larsbojensen) [reply package.json:122]: I will try to uninstall it again it and see if things still work

File: package.json (hunk 2 above)
Comment (tradeshift-larsbojensen) [reply package.json:115]: I cannot remove postcss as a direct dependency now without breaking at least the `build` script

File: package.json (hunk 1 above)
Comment (tradeshift-larsbojensen) [reply package.json]: If the checks also succeed, it does work without underscore as a direct dependency 👍

**Pull request merged**